EmailAddressFinder/src/main/java/com/mycompany/emailaddressfinder/EmailAddressFinder.java

```diff
@@ -0,0 +1,53 @@
+package com.mycompany.emailaddressfinder;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+public class EmailAddressFinder {
+
+    private Set<String> emailAddresses = new HashSet();
+
+    public static void main(String[] args) throws Exception {
+        EmailAddressFinder finder = new EmailAddressFinder();
+        finder.findEmailsOnDomain("http://www.iana.org");
```

Comment:
That is right. I kept it in for testing. I will push a newer version that uses the command line argument tonight.

Sent from my BlackBerry 10 smartphone on the Sprint network.
From: nikka-jana
Sent: Thursday, June 4, 2015 5:58 PM
To: lordniykon/simple-email-finder
Reply To: lordniykon/simple-email-finder
Cc: lordniykon
Subject: Re: [simple-email-finder] Initial Push (#1)


> +import java.util.HashSet;
> +import java.util.Set;
> +import java.util.regex.Matcher;
> +import java.util.regex.Pattern;
> +import org.jsoup.Jsoup;
> +import org.jsoup.nodes.Document;
> +import org.jsoup.nodes.Element;
> +import org.jsoup.select.Elements;
> +
> +public class EmailAddressFinder {
> +
> +    private Set<String> emailAddresses = new HashSet();
> +
> +    public static void main(String[] args) throws Exception {
> +        EmailAddressFinder finder = new EmailAddressFinder();
> +        finder.findEmailsOnDomain("http://www.iana.org");

Hmm, I believe one of the requirements was that the CLI tool takes the domain name as an argument.

---
Reply to this email directly or view it on GitHub:
https://github.com/lordniykon/simple-email-finder/pull/1/files#r31770157